src/isValidQuery.js

```diff
@@ -0,0 +1,6 @@
+const minMax = new RegExp(/(\()(?:[a-z][a-z]+)(-)((?:[a-z][a-z]+))?(-)?((?:[a-z][a-z]+))(:)(\s?)(\d+)((?:[a-z][a-z]+))\)/)
```

Comment:
Yeah, I didn't see a straightforward way to validate a query via the API but I'll keep looking. Thank!